tests/utils/test__select_column.py

```diff
@@ -320,7 +344,7 @@ def test_regex(df1):
     assert _select_column_names(re.compile(r"\d$"), df1) == list(
         df1.filter(regex=r"\d$").columns
     )
-    assert _select_column_names(patterns(r"\d$"), df1) == list(
+    assert _select_column_names(regex(r"\d$"), df1) == list(
         df1.filter(regex=r"\d$").columns
     )
```

Comment:
Taking the above comment into account, in which case, I suppose this test would be rendered useless (it's essentially a duplicate of the test above, yeah?)